framework/lib/components/text-field/formatted-number-input.tsx

```diff
@@ -78,46 +79,65 @@ export const FormattedNumberInput = ({
   numberOfDecimals = 2,
   max = Infinity,
   min = -Infinity,
+  inputLeftElement,
+  inputRightElement,
   ...rest
 }: FormattedNumberInputProps) => {
   const props = presetMap[preset]
+  const [ v, setV ] = useState(value)
+
+  const isInputValid = () => {
```

Comment:
Shouldn't this be called formatInput or something?

Reply:
Well its not really formatting, it is just validating that the input is within range, I renamed it to validateRange